mailer.js

```diff
@@ -311,11 +299,11 @@ module.exports = function (log) {
       subject: gettext('Your Firefox Account password has been changed'),
       template: 'passwordChangedEmail',
       templateValues: {
+        passwordChangeLinkAttributes: this._initiatePasswordChange(message.email),
```

Comment:
If a user's account password was changed, and it was not authorized, then they wouldn't know the new password to change it. The change password screen requires you to enter the current password and new password.

Would it make more sense to reset the password? This might be out of scope for this PR.